src/pages/patientView/trialMatch/TrialMatchTableUtils.spec.tsx

```diff
@@ -0,0 +1,294 @@
+                        ]
+                    },
+                    {
+                        "genomicAlteration": "!EGFR !L833V",
+                        "matches": [
+                            {
+                                "trueHugoSymbol": null,
+                                "trueProteinChange": null,
+                                "sampleIds": [ "P-0000628-T01-IM3", "P-0000628-T02-IM5" ]
+                            }
+                        ]
+                    }
+                ]
+            },
+            expected: 2
+        }, {
+            clinicalMatchGroup: {
+                "trialAgeNumerical": ">=18",
+                "trialOncotreePrimaryDiagnosis": {
+                    "general": [ "Non-Small Cell Lung Cancer" ],
+                    "not": []
+                },
+                "matches": [
+                    {
+                        "genomicAlteration": "EGFR 729_761del",
+                        "matches": [
+                            {
+                                "trueHugoSymbol": "EGFR",
+                                "trueProteinChange": "L747_P753delinsS",
+                                "sampleIds": [ "P-0000628-T01-IM3" ]
+                            }
+                        ]
+                    }
+                ],
+                "notMatches": [
+                    {
+                        "genomicAlteration": "!EGFR !T790M",
+                        "matches": [
+                            {
+                                "trueHugoSymbol": null,
+                                "trueProteinChange": null,
+                                "sampleIds": [ "P-0000628-T01-IM3" ]
+                            }
+                        ]
+                    }
+                ]
+            },
+            expected: 1
+        }, {
+            clinicalMatchGroup: {
+                "trialAgeNumerical": ">18",
+                "trialOncotreePrimaryDiagnosis": {
+                    "general": [ "Lung Adenocarcinoma" ],
+                    "not": []
+                },
+                "matches": [
+                    {
+                        "genomicAlteration": "EGFR Oncogenic Mutations",
+                        "matches": [
+                            {
+                                "trueHugoSymbol": "EGFR",
+                                "trueProteinChange": "L747_P753delinsS",
+                                "sampleIds": [ "P-0000628-T01-IM3", "P-0000628-T02-IM5" ]
+                            },
+                            {
+                                "trueHugoSymbol": "EGFR",
+                                "trueProteinChange": "T790M",
+                                "sampleIds": [ "P-0000628-T02-IM5" ]
+                            }
+                        ]
+                    }
+                ],
+                "notMatches": []
+            },
+            expected: 0
+        } ];
+        _.forEach(data, (item) => {
```

Comment:
probably better to avoid include the expected value in data structure above.  if there were a lot of assertions, maybe, but for only three, not important.  you could also then put a failure message as third param of assert, which gives better indication of the failure.

Reply:
Stored expect value in another array.